src/tiledb/cloud/utilities/_common.py

```diff
@@ -356,7 +363,16 @@ def wrapper(*args, **kwargs) -> Dict[str, object]:
             mode=dag.Mode.BATCH,
         )
 
-        # Submit the function as a batch UDF
+        # Guard against Node constructor peeling off "resources" or
+        # other functions peeling off "namespace" and "credentials".
+        # The "carry along" pattern was introduced in gh-630.
+        kwargs["carry_along"] = {
+            "resources": resources,
+            "namespace": namespace,
+            "access_credentials_name": acn,
+        }
```

Comment:
It looks to me that we need to extend the "carry along" pattern all the way into `as_batch()` so that `graph.submit()` below doesn't lose the resources parameter.